ISSUE_TEMPLATE/feature_time.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Feature time
```

Comment:
I'm nitpicking here, but why "time"? I can't help up think of Adventure Time when I read this. Why not Feature request?

Reply:
So sorry about not responding sooner. Somehow the notification for this slipped though the cracks for me. I updated this in a new PR.